sqlx-cli/src/prepare.rs

```diff
@@ -23,6 +23,12 @@ pub fn run(url: &str, merge: bool, cargo_args: Vec<String>) -> anyhow::Result<()
         data: QueryData,
     }
 
+    anyhow::ensure!(
+        Path::new("Cargo.toml").exists(),
+        r#"Failed to read `Cargo.toml`.
+hint: This command only works in the manifest directory of a Cargo package."#
+    );
```

Comment:
Both `run` and `check` call `run_prepare_step` which then executes some `cargo` command requiring `Cargo.toml` to be present. Would it make more sense to move the check to `run_prepare_step`?

Reply:
Yeah, I agree that makes more sense. I'll move the check to the beginning of `run_prepare_step`.